docs/GettingStarted.md

```diff
@@ -722,9 +722,10 @@ overhead.
 
 Priority sampling consists in deciding if a trace will be kept by using a priority attribute that will be propagated for distributed traces. Its value gives indication to the Agent and to the backend on how important the trace is.
 
-* 0: Don’t keep the trace.
+* -1: The user asked not to keep the trace.
```

Comment:
I think that part is very complicated for new users. I would prefer having constants (already suggested) and the snippet as example. In that way we remove any concern about "what is the number I have to use now".